examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -99,7 +102,8 @@ void CHIPCommand::StopTracing()
 void CHIPCommand::SetIdentity(const char * identity)
 {
     std::string name = std::string(identity);
-    if (name.compare(kIdentityAlpha) != 0 && name.compare(kIdentityBeta) != 0 && name.compare(kIdentityGamma) != 0)
+    if (name.compare(kIdentityAlpha) != 0 && name.compare(kIdentityBeta) != 0 && name.compare(kIdentityGamma) != 0 &&
+        name.compare(kIdentityNull))
```

Comment:
Doesn't that last check need `!= 0` at the end?

Reply:
Indeed it does.

Reply:
Fixed.